util/build_configs/cray-internal/setenv-hpe-cray-ex-x86_64.bash

```diff
@@ -340,9 +334,9 @@ else
         list_loaded_modules
     fi
 
-    #[TODO] gen_version_gcc=
-    #[TODO] gen_version_intel=
-    #[TODO] gen_version_cce=
+    gen_version_gcc=8.3.0
+    gen_version_intel=16.0.3.210
```

Comment:
Can we leave this as a TODO if we're not building with intel yet?

Reply:
(I do not feel strongly about this btw, I'm fine with leaving this in)

Reply:
Yeah, I might have gotten a bit over-enthused there.  Guess I could re-TODO it.